x-pack/auditbeat/module/system/login/login.go

```diff
@@ -11,15 +11,16 @@ import (
 	"net"
 	"time"
 
+	"github.com/elastic/beats/v7/auditbeat/ab"
 	"github.com/elastic/beats/v7/auditbeat/datastore"
 	"github.com/elastic/beats/v7/libbeat/common/cfgwarn"
 	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/beats/v7/x-pack/auditbeat/module/system"
 	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/elastic/elastic-agent-libs/mapstr"
 )
 
 const (
-	moduleName    = "system"
```

Comment:
I think all of these related module name changes should be reverted now that you have added a separate module registry.

Reply:
Really? This way is much cleaner using a single constant, instead of the same one in each module.

Not changing any behavior just more of a cleanup now. Seems like throwing away cleaner code.

Reply:
True. I was more interested in reducing the changeset to only include the necessary changes to make it more digestible to reviewers.